### PR TITLE
V1/2_POD_IP error when there are other pods labelled version=v1/2

### PR DESCRIPTION
**Pull request by SerenaFeng**
When there are multiple pods labelled version=v1, the V1_POD_IP is error, and so does V2_POD_IP. In my case for example, except httpbin, bookinfo example is also deployed in the cluster, which includes some pods labelled as version=v1, the result is like:

```
$ k g po
NAME                              READY   STATUS    RESTARTS   AGE
details-v1-78d78fbddf-fvgn5       2/2     Running   0          21h
httpbin-v1-6b4c749d6-cvk5r        2/2     Running   0          20h
httpbin-v2-5748c488bc-k7bwk       2/2     Running   0          20h
productpage-v1-596598f447-pmndz   2/2     Running   0          21h
ratings-v1-6c9dbf6b45-cchmb       2/2     Running   0          21h
reviews-v1-7bb8ffd9b6-2ljlx       2/2     Running   0          21h
reviews-v2-d7d75fff8-6p48p        2/2     Running   0          21h
reviews-v3-68964bc4c8-hxgsk       2/2     Running   0          21h
sleep-bb596f69d-wdlrj             2/2     Running   0          21h
$ export V1_POD_IP=$(kubectl get pod -l app=httpbin -l version=v1 -o jsonpath={.items..status.podIP})
$ echo $V1_POD_IP
10.244.50.74 10.244.193.205 10.244.50.78 10.244.50.75 10.244.193.204
```

After fixing it as the patch method, the result would be

```
$ export V1_POD_IP=$(kubectl get pod -l app=httpbin,version=v1 -o jsonpath={.items..status.podIP})
$ echo $V1_POD_IP
10.244.193.205
```

### CODE REVIEW

File: content/en/docs/tasks/security/dns-cert/index.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ and that the certificate contains the configured DNS names, you need to get the
 decode it, and view its text output with the following command:
 
 {{< text bash >}}
-$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in /dev/stdin -text -noout
```
Comment (frankbu) [inline content/en/docs/tasks/security/dns-cert/index.md]: Why is this change needed? Why can't `-` be used here?

Comment (SerenaFeng) [reply content/en/docs/tasks/security/dns-cert/index.md]: I think I made a mistake, this pull request is based on the master branch, soon after I submitted another patch with the dns-cert/index.md file changed to the master branch, it is also appeared here, I have submitted a new patch based on the newly created branch https://github.com/istio/istio.io/pull/6282, I will close this one soon.

**Comment by SerenaFeng** [comment]
submit a new pull request #6282 based a newly created branch other than the master branch
